indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -268,8 +284,10 @@
                                        nil)))
                              (remove nil?))
         humanized-fields (map #(humanizer/collection-humanizers-elastic context %) parsed-concepts)
-        suggestion-docs (map get-suggestion-docs humanized-fields)]
-    (flatten suggestion-docs)))
+        suggestion-docs (->> (map get-suggestion-docs humanized-fields)
+                             flatten
+                             (remove anti-value-suggestion?))]
```

Comment:
Nitpick: I always find it easier to read threaded statements when you thread a bound value rather than an expression
```suggestion
        suggestion-docs (->> humanized-fields
                             (map get-suggestion-docs)
                             flatten
                             (remove anti-value-suggestion?))]
```